src/scss/_layout-modern.scss

```diff
@@ -12,16 +12,26 @@
   }
 
   .ucf-event-title-wrapper {
-    margin-bottom: .25rem;
+    margin-bottom: .4rem;
   }
 
   .ucf-event-title {
     color: $white;
+    font-size: 1.2rem;
     font-weight: 700;
   }
 
-  .ucf-event-location {
-    color: $white;
+  .ucf-event-description-wrapper {
+    font-size: .9rem;
+
+    a {
+      color: $white;
```

Comment:
I'd recommend letting text color (white and gold text in particular) get set at the theme level instead of in the plugin--in case events are ever displayed against light backgrounds on other sites.